src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -188,26 +186,22 @@ class SidebarLinks extends React.Component {
                         </OfflineWithFeedback>
                     </TouchableOpacity>
                 </View>
-                <Freeze
-                    freeze={shouldFreeze}
-                    placeholder={skeletonPlaceholder}
-                >
```

Comment:
Sure, will do!

Reply:
I will go thought the files in depth but just a quick response. I assume LHNOptionList is exactly the same and the only difference is Freeze is that right? Then It would be better to isolate that component instead as @neil-marcellini suggested. **BUT** why not remove the Freeze component, is it really doing something significantly good? It's anti react's life-cycle and we will remove it on navigation reboot anyway.

Reply:
Just realised this is a part of navigation reboot :grin: I think It was planned to remove Freeze at this point, why we are still using it. Am I missing something here? cc @neil-marcellini @mountiny (and @roryabraham I guess you know about this)